demos/storybook/stories/drawer/with-multiple-DrawerNavGroups.tsx

```diff
@@ -7,16 +7,22 @@ import React from 'react';
 import { DrawerStoryContext } from './util';
 import { navGroupItems1 } from './with-basic-config';
 
-export const navGroupItems2: NavItem[] = [
+export const navGroupItems2 = (context: DrawerStoryContext): NavItem[] => [
     {
         title: 'Overview',
         itemID: 'group2-1',
         icon: <Dashboard />,
+        onClick: (): void => {
+            context.store.set({ selected: 'group2-1' });
```

Comment:
Is setting the selected item going to break the context in stories that don't have a second navgroup?

Reply:
No. It will just have everything unselected.